zerver/data_import/slack.py

```diff
@@ -314,7 +322,13 @@ def get_user_email(user: ZerverFieldsT, domain_name: str) -> str:
             slack_bot_name = user['profile']['first_name']
         else:
             raise AssertionError("Could not identify bot type")
-        return slack_bot_name.replace("Bot", "").replace(" ", "") + f"-bot@{domain_name}"
+        bot_email = slack_bot_name.replace("Bot", "").replace(" ", "") + f"-bot@{domain_name}"
+        if bot_map is not None:
```

Comment:
Is there a reason `bot_map` would ever be `None` here?  I think you can omit this condition and just add `assert(bot_map is not None)`.  You probably don't even need the assert here.

Reply:
I had added that condition because I have taken <code>bot_id</code> and <code> bot_map</code> as an optional parameter whose default value is <code>None</code> .Should I change this to mandatory parameter?

Reply:
Yeah, if it'll always be present, we should make it a mandatory parameter.